client/src/components/Landing/GetStartedModal.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react'
```

Comment:
Since we already have a 'SignUpModal' component built, does the landing page need its own separate modal? It might make more sense to re-use the already existing modal which already has this functionality.

Reply:
The get started modal was removed and replaced with the sign up modal.